ement-room.el

```diff
@@ -3450,6 +3456,12 @@ For use in `completion-at-point-functions'."
                         collect (ement-room-id room)
                         collect (ement-room-canonical-alias room))))))
 
+;;;;; Reaction
+
+(defun ement-room-reaction--read-char ()
```

Comment:
Let's also call the function `ement-read-reaction`, and let's put it in `ement.el` among the other functions.  It could eventually be used outside of `ement-room` buffers.